docs/tutorials/my-first-custom-action.rst

```diff
@@ -0,0 +1,255 @@
+      labels:
+        env: test
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
+        imagePullPolicy: IfNotPresent
+    #  nodeSelector:
+    #    spoiler: alert
+
+
+Automating the detection with a Robusta Playbook
+----
+What we need to do?
+----
+
+.. note::
+    Make sure to clean up the pod from the last section by running ``kubectl delete pod nginx``
+
+A playbook consists of two things:
+
+- Trigger - We’re gonna use a built in trigger
+- Action - We’re gonna write our own action!
+
+
+Finding the correct trigger
+----
+So what is the correct trigger for the job?
+We can think of two triggers that may fit:
+
+- Creation of a new pod (because we create a new pod, ‘ngnix’)
+- A Kubernetes Event is fired (because we ran kubectl get event to find out the scheduling error)
+
+Let’s look at the Trigger section about :ref:`Kubernetes (API Server)`, and try to find out triggers for both.
+Go ahead and try to find them!
+
+Okay! We find ``on_pod_create`` and ``on_event_create``
+
+We’ll use ``on_event_create``, because in this case ``on_pod_create`` is not even called because the pod is not created.
+
+Writing the action
+----
+
+Now we need to write code that checks this event and reports it. To find out the correct event class that matches our trigger ``on_event_create``. please take a look at :ref:`Event Hierarchy`.
+
+Okay! We find out it’s ``EventEvent``!
+
+So we need to get the information, check for the scenario, and then report it (for more information about reporting it see :ref:`Findings API`)
+
+Let’s name our action ``report_scheduling_failure``, and write everything in a python file:
+
+.. code-block:: python
+
+    from robusta.api import *
+
+    @action
+    def report_scheduling_failure(event: EventEvent):
+        actual_event = event.get_event()
+
+        print(f"This print will be shown in the robusta logs={actual_event}")
+
+        if actual_event.type.casefold() == f'Warning'.casefold() and \
+            actual_event.reason.casefold() == f'FailedScheduling'.casefold() and \
+            actual_event.involvedObject.kind.casefold() == f'Pod'.casefold():
+            _report_failed_scheduling(event, actual_event.involvedObject.name, actual_event.message)
+
+    def _report_failed_scheduling(event: EventEvent, pod_name: str, message: str):
+        # this is how you send data to slack or other destinations
+        event.add_enrichment([
+            MarkdownBlock(f"Failed to schedule a pod named '{pod_name}', error: {message}"),
+        ])
+
+Before we proceed, we need to enable local playbook repositories in Robusta.
+
+Follow this quick guide to learn how to package your python file for Robusta: :ref:`Custom playbook repositories`
+
+Use this useful debugging commands to make sure your action ( ``report_scheduling_failure``) is loaded:
+
+.. code-block:: bash
+
+    robusta logs # get robusta logs, see errors.
+    robusta playbooks list-dirs  # get see if you custom action was loaded
```

Comment:
I think it's more accurate to say: "# get see if you custom action package was loaded"
`list-dirs` display loaded packages I think